introduction.md

```diff
@@ -1,3 +1,3 @@
 Hello! I'm here to teach people about Git
-I like JavaScript and  HTML & CSS
+I like * JavaScript * HTML * CSS
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?